internal/app/singularity/cache_list_linux.go

```diff
@@ -15,25 +15,31 @@ import (
 	"github.com/sylabs/singularity/internal/pkg/cache"
 )
 
+const (
+	KiB = 1024
+	MiB = KiB * 1024
+	GiB = MiB * 1024
+	TiB = GiB * 1024
+)
+
 // findSize takes a size in bytes and converts it to a human-readable string representation
 // expressing kB, MB, GB or TB (whatever is smaller, but still larger than one).
```

Comment:
It's a nit-pick but shouldn't you change the comment to use KiB, MiB, GiB and TiB as well?

Reply:
Yep - that would have been smart... two sets of eyes missed it 🤦